airbyte-cdk/python/airbyte_cdk/sources/declarative/requesters/paginators/strategies/stop_condition.py

```diff
@@ -42,8 +42,12 @@ def next_page_token(self, response: requests.Response, last_records: List[Record
             return None
         return self._delegate.next_page_token(response, last_records)
 
-    def reset(self):
+    def reset(self) -> None:
         self._delegate.reset()
 
     def get_page_size(self) -> Optional[int]:
         return self._delegate.get_page_size()
+
+    @property
+    def initial_token(self) -> Optional[Any]:
```

Comment:
Why is this change needed?

Reply:
the method is an `abstractmethod` in the parent `PaginationStrategy` class so it must be implemented by `StopConditionPaginationStrategyDecorator`

Reply:
Okay, this is just something that was overlooked previously?